### PR TITLE
[lbaas2]: rewording changes

### PR DESCRIPTION
**Pull request by ArtieReus**
Added rewording changes proposed by @m-kratochvil

### CODE REVIEW

**Comment by ArtieReus** [comment]
@notandy, @kayrus and @m-kratochvil would you please review these changes as proposed in the Jira ticket?

File: plugins/lbaas2/app/javascript/app/components/healthmonitor/NewHealthMonitor.js [hunk 1]
```diff
@@ -166,12 +166,12 @@ const NewHealthMonitor = (props) => {
             />
             <span className="help-block">
               <i className="fa fa-info-circle"></i>
-              The maximum time, in seconds, that a monitor waits to connect
-              before it times out. This value must be less than the delay value.
+              The time, in seconds, after which a single health check probe
+              times out (fails). This value must be less than the delay value.
```
Comment (m-kratochvil) [inline plugins/lbaas2/app/javascript/app/components/healthmonitor/NewHealthMonitor.js:170]: replace `delay` with `interval`

**Review by m-kratochvil — CHANGES_REQUESTED** [review]

File: plugins/lbaas2/app/javascript/app/components/healthmonitor/EditHealthMonitor.js [hunk 2]
```diff
@@ -220,13 +223,13 @@ const EditHealthMonitor = (props) => {
                   />
                   <span className="help-block">
                     <i className="fa fa-info-circle"></i>
-                    The maximum time, in seconds, that a monitor waits to
-                    connect before it times out. This value must be less than
-                    the delay value.
+                    The time, in seconds, after which a single health check
+                    probe times out (fails). This value must be less than the
+                    delay value.
```
[the remark this answers is not in the document]
Comment (ArtieReus) [reply plugins/lbaas2/app/javascript/app/components/healthmonitor/EditHealthMonitor.js]: good catch!

File: plugins/lbaas2/app/javascript/app/components/healthmonitor/NewHealthMonitor.js (hunk 1 above)
Comment (ArtieReus) [reply plugins/lbaas2/app/javascript/app/components/healthmonitor/NewHealthMonitor.js]: good catch!

**Review by m-kratochvil — APPROVED** [review]
approved on my side, looks ok

**Pull request merged**